src/_shared/scss/_advert.scss

```diff
@@ -109,6 +114,7 @@
 
     &.u-text-hyphenate {
         max-width: 100%;
+        word-wrap: break-word;
```

Comment:
Yeah we should probably copy the `u-text-hyphenate` class from frontend in _core.scss